resources/sdk/clisdkclient/extensions/config/config.go

```diff
@@ -128,6 +139,10 @@ func applyEnvironmentVariableOverrides() {
 	if clientSecret != "" && ClientSecret == "" {
 		ClientSecret = clientSecret
 	}
+	accessToken := os.Getenv("GENESYS_CLOUD_ACCESS_TOKEN")
```

Comment:
There should be no underscore between genesys and cloud like with the other environment variables